user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md

```diff
@@ -0,0 +1,150 @@
+---
+uid: Register_Catalog_Item
+---
+
+# Registering a Catalog item
+
+## About Catalog items
+
+A Catalog item is a small package that contains one or more artifacts that can be deployed on a DataMiner System. [Different types of Catalog items](#supported-catalog-item-types) are supported. For example, an item could be a single connector, or it could be an application package containing multiple dashboards, Automation scripts, and connectors.
+
+### Public and private Catalog items
+
+Public Catalog items are items that are available for anyone within the community to use.
+
+Private Catalog items are items that are only available within your own organization. This gives users the possibility to develop app packages, Automation scripts or any type of Catalog item and redeploy them within their organization on various systems.
+
+### Versioning of Catalog items
+
+Catalog items can have multiple versions. To make sure that the versioning of items is easy to understand for everyone, [Semver](https://semver.org/) versioning is enforced. Users can assign extra labels to versions if they want to indicate that a certain version is not an official release (e.g. 1.2.3-alpha).
+
+> [!IMPORTANT]
+> If the item is of type Connector, you will need to adhere to the [existing connector versioning](xref:ProtocolVersionSemantics).
+
+### Supported Catalog item types
+
+At present, the following types of Catalog items are supported:
+
+- Automation Script
+- Application Package
+- Connector
+- Visio
+
+## Registering a private Catalog item
+
+### Using the Catalog UI
+
+If you use the UI, you can only register a Catalog item. There is no UI yet to register a new version. This can only be done by using the [API](xref:Register_Catalog_Item#register-using-the-catalog-api).
+
+To register a new Catalog item, do the following:
+
+1. Go to the [Catalog](https://catalog.dataminer.services).
+
+1. Open the user menu by clicking the user icon in the top-right corner.
+
+   Here you will find the *Register item* button.
```

Comment:
Should users click this button? If so, this should be included as a step of its own, instead of an indented line.

Reply:
done